src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1123,7 +1126,20 @@ bool entitiesQuery
 
         /* Attributes part */
 
-        std::vector<BSONElement> queryAttrV = r.getField(ENT_ATTRS).Array();
+        std::vector<BSONElement> queryAttrV;
```

Comment:
Fixed in  09f7218 and 006b090 (two commits as I had to commit to switch to the branch of another PR)